### ISSUE TITLE
Upgrade neighborhoods and localities in greater Buenos Aires province

### ISSUE THREAD

**Issue by rmglennon** [whosonfirst-data/whosonfirst-data#896]
This is related to https://github.com/whosonfirst-data/whosonfirst-data/issues/180, but is for an area outside the city of Buenos Aires.

This came in through support (see Desk 846). The user was using Mapzen Search to find Florida, Buenos Aires (province), Argentina, but there are no relevant results.

https://en.wikipedia.org/wiki/Florida,_Buenos_Aires

Florida is a barrio (locality) in the partido of Vicente López within the province of Buenos Aires. 

Walking through the hierarchy, there are some missing descendants of Vicente López, including Florida Este and Florida Oeste. https://whosonfirst.mapzen.com/spelunker/id/1108718133/descendants/?exclude=nullisland

It appears that only the city of Buenos Aires is included in the data from http://www.buenosaires.gob.ar/laciudad/barrios (Vicente López is at the northern border). And, therefore, I don't think this will be fixed in https://github.com/whosonfirst-data/whosonfirst-data/pull/881. Please close if that pull request contains broader data, or if this is duplicated with another issue.

Unfortunately, I did not see an obvious source of data for this area.

**Comment by stepps00** [comment]
## tl;dr

Status:
- [ ] Propose **sources**
- [ ] Establish **concordances** between WOF and source
- [ ] **Upgrade / add** WOF shapes
- [ ] **Add new or missing** features to WOF
- [ ] **Deprecate / supersede** WOF features
- [ ] **Upgrade** locality record's shape
- [ ] **Draft review**
- [ ] **Draft revisions** based on review comments
- [ ] **Final review**
- [ ] **Final revisions** based on review comments
- [ ] **Import** into Who's On First

### Details

Sourcing:
- Permissive license?
- Add new src to Who's On First

Upgrade / add:
- `macrohood`
- `neighbourhood`
- `microhood`

Add new or missing features to WOF
- `macrohood`
- `neighbourhood`
- `microhood`

Deprecate / supersede:
- `macrohood`
- `neighbourhood`
- `microhood`
